src/dhanshri/Assignment_12/StaticNonstaticReturnType.java

```diff
@@ -0,0 +1,55 @@
+package dhanshri.Assignment_12;
+
+public class StaticNonstaticReturnType {
+
+	double sum(double x, double y, double z) {
+		double answer = x + y + z;
+		return answer;
+
+	}
+
+	double average(double sumreturn) {
+		double avgAns = sumreturn / 3;
+		return avgAns;
+
+	}
+
+	boolean isAnswerAboveExpecation(double averageReturn) {
+		if (averageReturn > 50)
+			return true;
+		else {
+			return false;
+		}
+	}
+
+	static boolean isEligible(double avgexception) {
+		if (avgexception > 50 || avgexception % 2 == 0)
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.